frontend/js/src/artist/ArtistPage.tsx

```diff
@@ -41,21 +42,26 @@ export type ArtistPageProps = {
 
 export default function ArtistPage(): JSX.Element {
   const { APIService } = React.useContext(GlobalAppContext);
+  const location = useLocation();
+  const { artistMBID } = useParams() as { artistMBID: string };
   const {
-    artist,
-    popularRecordings,
-    releaseGroups,
-    similarArtists,
-    listeningStats,
-    coverArt: coverArtSVG,
-  } = useLoaderData() as ArtistPageProps;
+    data: {
+      artist,
```

Comment:
I deployed to test.LB but found that this line was throwing an error when I moved from one artist to another.
To be clear, loading an artist page works, but if I click on one of the similar artists at the bottom of the page an error is thrown because  "cannot read property artist of undefined" since `data` is apparently undefined.

Reply:
Fixed now